pkgs/tools/package-management/nix-universal-prefetch/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, fetchFromGitHub
+, ruby
+}:
+
+# No gems used, so mkDerivation is fine.
+stdenv.mkDerivation rec {
+  name = "nix-universal-prefetch-${version}";
```

Comment:
Support for `pname` and `version` [has been added to `stdenv.mkDerivation`](https://github.com/NixOS/nixpkgs/blob/c8c53fcb1154999dabac350bebe12611a0f75024/pkgs/stdenv/generic/make-derivation.nix#L84) (it is not backported), so considering you already do add `version`, you should probably split `name` in `pname` as well.